frontend/src/assets/css/fonts.scss

```diff
@@ -0,0 +1,143 @@
+@font-face {
```

Comment:
Let's leave only 600/400 without italic as we have on our figma design, so that we don't load fonts that we don't use